modules/apps/remote-app/remote-app-web/src/main/java/com/liferay/remote/app/web/internal/portlet/action/EditRemoteAppEntryMVCActionCommand.java

```diff
@@ -96,11 +96,16 @@ private void _add(ActionRequest actionRequest) throws PortalException {
 		String type = ParamUtil.getString(actionRequest, "type");
 
 		if (type.equals(RemoteAppConstants.TYPE_CUSTOM_ELEMENT)) {
+			String[] customElementCSSURLs = ParamUtil.getStringValues(
```

Comment:
Brian is in everything-should-be-inlined mode so maybe he inlines this, but I prefer with a explicit variable and readable method calls and is not consistent with it, so I'll leave it this way, just FYI.

Reply:
Yeah, hard to know... I just copied this from the `_update` method down below in this same file, so either this is kept like this, or both should be updated 🤷